Enterprise-CMCS/managed-care-review
# [MCR-2613] Edit MC-CRS number

## Summary

to be merged after https://github.com/Enterprise-CMCS/managed-care-review/pull/1996

This PR adds test relevant to editing a MC-CRS record number

#### Related issues

https://qmacbis.atlassian.net/browse/MCR-2613

#### Screenshots

<img width="792" alt="Screenshot 2023-10-25 at 10 57 54 AM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/ef1502c8-db56-4f39-9d95-a2082193ef65">
<img width="967" alt="Screenshot 2023-10-25 at 10 58 02 AM" src="https://github.com/Enterprise-CMCS/managed-care-review/assets/67110378/94130c0b-c003-43e0-99a6-7af52b9a28e5">


#### Test cases covered

- MC-CRS id pre-populates on the text field if the `mccrsID` is set
- Displays the Edit MC-CRS id link

## QA guidance

- Sign in as a CMS user 
- Click on a submission and add a MC-CRS id. Once back on the submission summary page, click Edit MC-CRS number and you should be taken back to the text field with the previous value pre-populated
